plugins/jira-dashboard-backend/src/service/router.test.ts

```diff
@@ -49,6 +58,7 @@ describe('createRouter', () => {
       discovery: testDiscovery,
       identity: { getIdentity },
       tokenManager,
+      userInfo: userInfo
```

Comment:
We can use "mockServices" from "@backstage/backend-test-utils" here. 

`
import { mockServices } from '@backstage/backend-test-utils';
 userInfo: mockServices.userInfo({ userEntityRef: 'user:default/guest' }),
`

Reply:
code updated